src/lib/fcitx/text.cpp

```diff
@@ -55,6 +55,14 @@ void Text::append(std::string str, TextFormatFlags flag) {
     d->texts_.emplace_back(std::move(str), flag);
 }
 
+void Text::append(const Text &text) {
```

Comment:
Can we make this function accept Text, so all the std::string could be moved from?